### PR TITLE
feat: migrate whole toppanel

### PR DESCRIPTION
**Pull request by de-lan**
## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [x] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19053](https://cdap.atlassian.net/browse/CDAP-19053)

## Test Plan
Manul + e2e

### CODE REVIEW

File: app/cdap/components/PostRunActions/PostRunActionsWizard.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export default class PostRunActionsWizard extends Component<IPostRunActionsWizar
           };
         });
       } catch (e) {
-        console.log(`Cannot parse widget JSON for ${action.plugin.name}`, e);
+        // console.log(`Cannot parse widget JSON for ${action.plugin.name}`, e);
```
Comment (seanfeld) [inline app/cdap/components/PostRunActions/PostRunActionsWizard.tsx]: uncomment this

File: app/cdap/components/PostRunActions/WizardNav/index.tsx [hunk 2]
```diff
@@ -117,7 +117,7 @@ export const WizardNav = (props: IWizardNavProps) => {
         });
         props.setAction(action);
       } catch (e) {
-        console.log(`Cannot parse widget JSON for ${action.plugin.name}`, e);
+        // console.log(`Cannot parse widget JSON for ${action.plugin.name}`, e);
```
Comment (seanfeld) [inline app/cdap/components/PostRunActions/WizardNav/index.tsx]: remove this comment too

File: app/cdap/components/PostRunActions/WizardNav/index.tsx [hunk 3]
```diff
@@ -181,7 +181,7 @@ export const WizardNav = (props: IWizardNavProps) => {
           setPostActionsList(filterPlugins(res));
         },
         error: (err) => {
-          console.log('ERROR: ', err);
+          // console.log('ERROR: ', err);
```
Comment (seanfeld) [inline app/cdap/components/PostRunActions/WizardNav/index.tsx]: ?

Comment (seanfeld) [reply app/cdap/components/PostRunActions/WizardNav/index.tsx]: if this isn't passing validation then add the exception comment for console log
```
    // tslint:disable-next-line: no-console
    ```

File: app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx [hunk 4]
```diff
@@ -356,8 +352,8 @@ export const ActionButtons = ({
         applyBatchConfig={applyBatchConfig}
         applyRealtimeConfig={applyRealtimeConfig}
         actionCreator={actionCreator}
-        isDeployed={isDeployed}
-        showPreviewConfig={showPreviewConfig}
+        isDeployed={false}
```
Comment (seanfeld) [inline app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx:355]: this is always false now?

File: app/hydrator/templates/create/reacttoppanel.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  get-schedule-Info="HydratorPlusPlusTopPanelCtrl.getScheduleInfo"
+  action-creator="HydratorPlusPlusTopPanelCtrl.HydratorPlusPlusConfigActions"
+  apply-runtime-arguments="HydratorPlusPlusTopPanelCtrl.applyRuntimeArgumentsFromReactStore"
+  apply-batch-config="HydratorPlusPlusTopPanelCtrl.applyBatchConfigFromReactStore"
+  apply-realtime-config="HydratorPlusPlusTopPanelCtrl.applyRealtimeConfigFromReactStore"
+  get-post-actions="HydratorPlusPlusTopPanelCtrl.getPostActions"
+  validate-plugin-properties="HydratorPlusPlusTopPanelCtrl.validatePluginProperties"
+  get-runtime-args="HydratorPlusPlusTopPanelCtrl.getRuntimeArgumentsV2"
+  get-store-config="HydratorPlusPlusTopPanelCtrl.getStoreConfig"
+></top-panel-react>
```
Comment (seanfeld) [inline app/hydrator/templates/create/reacttoppanel.html]: new line

File: app/directives/react-components/index.js [hunk 6]
```diff
@@ -190,15 +190,10 @@ angular
...
+  .directive('topPanelReact', function(reactDirective) {
+    return reactDirective(window.CaskCommon.TopPanelReact);
   })
   .directive('pipelineCanvasActionBtns', function(reactDirective) {
     return reactDirective(window.CaskCommon.PipelineCanvasActionBtns);
-  })
-  .directive('pipelineConfigure', function (reactDirective) {
-    return reactDirective(window.CaskCommon.PipelineConfigure);
   });
+
```
Comment (seanfeld) [inline app/directives/react-components/index.js]: you can set your editor to always make a new line at the end of a file

**Review by seanfeld — APPROVED** [review]
just nits

File: app/cdap/components/PostRunActions/WizardNav/index.tsx (hunk 3 above)
Comment (de-lan) [reply app/cdap/components/PostRunActions/WizardNav/index.tsx]: gotcha, will add exceptions

File: app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx (hunk 4 above)
Comment (de-lan) [reply app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx:355]: yep, it was always passed in a false in toppanel configure page

File: app/cdap/components/PostRunActions/WizardNav/index.tsx [hunk 7]
```diff
@@ -78,7 +78,7 @@ export const WizardNav = (props: IWizardNavProps) => {
   function pluginFetch(action) {
     if (action) {
       const { name, version, scope } = action.plugin.artifact;
-      const params = {
+      const pluginParams = {
```
Comment (sumengwang) [inline app/cdap/components/PostRunActions/WizardNav/index.tsx:81]: Thank you for fixing this :)

**Review by sumengwang — APPROVED** [review]
Looks great

**Pull request merged**